sdcm/cluster.py

```diff
@@ -4254,22 +4254,19 @@ def verify_logging_from_nodes(nodes_list):
         return True
 
     @wait_for_init_wrap
-    def wait_for_init(self, node_list=None, verbose=False, timeout=None, check_node_health=True):  # pylint: disable=unused-argument, too-many-arguments
-        """
-        Scylla cluster setup.
-        :param node_list: List of nodes to watch for init.
-        :param verbose: Whether to print extra info while watching for init.
-        :param timeout: timeout in minutes to wait for init to be finished
-        :param check_node_health: select if run node health check or not
-        :return:
-        """
+    def wait_for_init(self, node_list=None, verbose=False, timeout=None, check_node_health=True):  # pylint: disable=unused-argument
         node_list = node_list or self.nodes
-        self.get_scylla_version()
-
-        wait.wait_for(self.verify_logging_from_nodes, nodes_list=node_list,
-                      text="wait for db logs", step=20, timeout=300, throw_exc=True)
-
-        self.log.info("{} nodes configured and started.".format(node_list))
+        wait.wait_for(
+            func=self.verify_logging_from_nodes,
+            step=20,
+            text="wait for db logs",
+            timeout=300,
+            throw_exc=True,
+
+            # self.verify_logging_from_nodes() parameters:
```

Comment:
Do we really need this commented line?

Reply:
I think so: `wait_for()` has too many own parameters and I decided to make it clear which parameters will go to the wrapped function.

Reply:
It is not really obvious, it looks like something incomplete.

Reply:
also we are not doing it anywhere else and it is perceived as some leftover and make you spend time to decipher what exactly it mean.

Reply:
Got it. Will comment line like 
```
# Following parameters will be passed to self.verify_logging_from_nodes():
```
works better for you?

Reply:
@enaydanov it doesn't solve the confusion. Comment must be self-sufficient.
Something like following:
```
NOTE: "node_list" param is needed for "verify_logging_from_nodes" method we wait for
```

Reply:
Looks like my mind works in a different way :) It doesn't worth to discuss it anymore, just removed it. Thanks.